src/3d/qgs3dsceneexporter.cpp

```diff
@@ -565,11 +622,40 @@ Qgs3DExportObject *Qgs3DSceneExporter::processGeometryRenderer( Qt3DRender::QGeo
     positionData = getAttributeData<float>( positionAttribute, vertexBytes );
   }
 
-//   For tessellated polygons that don't have index attributes
+  // For tessellated polygons that don't have index attributes
   if ( positionAttribute != nullptr && indexAttribute == nullptr )
   {
-    for ( int i = 0; i < positionData.size() / 3; ++i )
-      indexData.push_back( i );
+    for ( uint i = 0; i < ( uint ) positionData.size() / 3; ++i )
```

Comment:
Can you avoid the c style casts in this method?
Maybe adding temporary variables would help as static_casts sometimes make expressions look too long.

Reply:
done!